client/landing/jetpack-cloud/components/masterbar/index.jsx

```diff
@@ -0,0 +1,31 @@
+/**
+ * External dependencies
+ */
+import React from 'react';
+
+/**
+ * Internal dependencies
+ */
+import Item from 'layout/masterbar/item';
+import JetpackLogo from 'components/jetpack-logo';
+import Masterbar from 'layout/masterbar/masterbar';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function() {
+	return (
+		<Masterbar>
+			<Item
+				tipTarget="jetpack"
```

Comment:
I think this part can be removed for now. Since we are not using any guided tours.

Reply:
Sounds good. I removed it in d02df7b.